src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/AttendanceService.kt

```diff
@@ -27,7 +27,7 @@ open class AttendanceService(
     val log: Logger = LoggerFactory.getLogger(this::class.java)
   }
 
-  open fun getAttendanceForEventLocation(prisonId: String?, eventLocationId: Long?, date: LocalDate?, period: TimePeriod?): Set<AttendanceDto> {
+  fun getAttendanceForEventLocation(prisonId: String?, eventLocationId: Long?, date: LocalDate?, period: TimePeriod?): Set<AttendanceDto> {
```

Comment:
I know it's not this change, but I was just wondering if these parameters are really nullable or if it was just because of a Java conversion?